cluster-provision/k8s/1.25/provision.sh

```diff
@@ -102,7 +114,12 @@ baseurl=https://storage.googleapis.com/kubevirtci-crio-mirror/devel_kubic_libcon
 gpgcheck=0
 enabled=1
 EOF
-dnf install -y cri-o containers-common-1-23.module_el8.7.0+1106+45480ee0.x86_64
+if [[ "$release" == "centos8" ]]; then
+    dnf install -y cri-o containers-common-1-23.module_el8.7.0+1106+45480ee0.x86_64
+elif [[ "$release" == "centos9" ]]; then
+    dnf install -y cri-o
```

Comment:
cri-o will probably still fail to pull images if we don't do some sort of patch to `/etc/containers/policy.json` - I guess centos stream 9 has the newer version which cri-o is not able to parse correctly yet?

see https://github.com/cri-o/cri-o/issues/6197 for more details

Reply:
Ok this doesn't look to be an issue here - the containers-common from the cri-o mirror satisfies the requirements. 
```
[root@79482ac54e73 /]# dnf info containers-common-4:1-19.el8.28.9.noarch
Last metadata expiration check: 0:03:36 ago on Tue Nov 15 10:27:04 2022.
Installed Packages
Name         : containers-common
Epoch        : 4
Version      : 1
Release      : 19.el8.28.9
Architecture : noarch
Size         : 81 k
Source       : containers-common-1-19.el8.28.9.src.rpm
Repository   : @System
From repo    : devel_kubic_libcontainers_stable
Summary      : Common configuration and documentation for containers
License      : ASL 2.0
Description  : This package contains common configuration files and documentation for container
             : tools ecosystem, such as Podman, Buildah and Skopeo.
             :
             : It is required because the most of configuration files and docs come from projects
             : which are vendored into Podman, Buildah, Skopeo, etc. but they are not packaged
             : separately.

[root@79482ac54e73 /]# cat /etc/containers/policy.json
{
    "default": [
        {
            "type": "insecureAcceptAnything"
        }
    ],
    "transports":
        {
            "docker-daemon":
                {
                    "": [{"type":"insecureAcceptAnything"}]
                }
        }
}
```